detd/devices/device.py

```diff
@@ -42,15 +39,11 @@ def from_pci_id(pci_id):
 
     raise NameError("Unrecognized PCI ID: {}".format(pci_id))
 
-
-
-
 class Capability(enum.Enum):
     Qbv = 0
     Qbu = 1
     LTC = 2
-
-
+    Preemption = 3
```

Comment:
No need to specify it twice, as Preemption is already covered by Qbu (technically it should be 3br/Qbu, but let us keep it simple...).